routes/meetings.js

```diff
@@ -140,17 +160,22 @@ router
         .status(400)
         .json({ error: "There are no fields in the request body" });
     }
+
+    meetingPostData.title = xss(meetingPostData.title);
+    meetingPostData.textBody = xss(meetingPostData.textBody);
+    meetingPostData.tag = xss(meetingPostData.tag);
+
     //validation
     let errorMessages = utils.validateMeetingCreateInputs(
       meetingPostData.title,
-      meetingPostData.dateAddedTo,
-      meetingPostData.dateDueOn,
-      meetingPostData.priority,
+      xss(meetingPostData.dateAddedTo),
+      xss(meetingPostData.dateDueOn),
+      xss(meetingPostData.priority),
       meetingPostData.textBody,
       meetingPostData.tag,
-      meetingPostData.repeating,
-      meetingPostData.repeatingCounterIncrement,
-      meetingPostData.repeatingIncrementBy
+      xss(meetingPostData.repeating),
+      xss(meetingPostData.repeatingCounterIncrement),
+      xss(meetingPostData.repeatingIncrementBy)
```

Comment:
you need to store the xss values in the object and that needs to be passed to the function